### PR TITLE
Adding support for Flight Order Management API

### PR DESCRIPTION
**Pull request by akshitsingla**
## Changes for this pull request
#43 Adding support for Flight Order Management API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -300,6 +300,13 @@ amadeus.shopping.hotelOffersByHotel.get({
 // Confirm the availability of a specific offer id
 amadeus.shopping.hotelOffer('XXX').get()
 
+
+// Retrieve flight order with ID
```
Comment (tsolakoua) [inline README.md]: I would add `The flight ID comes from the Flight Create Orders (in test environment it's temporary)`

File: src/amadeus/namespaces/booking.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  constructor(client) {
+    this.client    = client;
+  }
+
+  flightOrders (orderId) {
+    return new FlightOrders(this.client, orderId);
+  }
+}
+
+export default Booking;
```
Comment (tsolakoua) [inline src/amadeus/namespaces/booking.js]: new line needed?

File: src/amadeus/namespaces/booking/flight_orders.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+   */
+  delete() {
+    if (this._orderId)
+      return this.client.delete('/v1/booking/flight-orders/' + this._orderId);
+    else
+      throw new Error('MISSING_REQUIRED_PARAMETER');
+  }
+}
+
+export default FlightOrders;
```
Comment (tsolakoua) [inline src/amadeus/namespaces/booking/flight_orders.js]: new line needed?

File: src/amadeus/namespaces/booking.js (hunk 2 above)
Comment (akshitsingla) [reply src/amadeus/namespaces/booking.js]: Nope.

File: src/amadeus/namespaces/booking/flight_orders.js (hunk 3 above)
Comment (akshitsingla) [reply src/amadeus/namespaces/booking/flight_orders.js]: Nope.

File: README.md (hunk 1 above)
Comment (akshitsingla) [reply README.md]: Fixed

**Comment by tsolakoua** [comment]
All good @akshitsingla, just the conflicts need to be resolved

**Pull request merged**